magento/inventory
# MSI-1622: The limit is 100 times less than in magento/magento2#2.3-develop.

### Description

1. JS validation has been added for Source Item's quantity field on a Product page.
2. `inventory_source_item` `quantity` column changed from `decimal(10, 4)` to `decimal(12, 4)`.

### Fixed Issues (if relevant)

1. magento-engcom/msi#1622: The limit is 100 times less than in magento/magento2#2.3-develop.

### Manual testing scenarios

1. Start creating a Simple Product and fill all the required fields.
2. Assign product to any source and Set the quantity to be equal to 100000000.
3. JS validation message should appear:
'Please enter a value less than or equal to 99999999.'


### Contribution checklist
